numberformat/diff.emu

```diff
@@ -1472,6 +1472,12 @@
           1. Let _v_ be the value of _nf_'s internal slot whose name is the Internal Slot value of the current row.
           1. If _v_ is not *undefined*, then
             1. Perform ! CreateDataPropertyOrThrow(_options_, _p_, _v_).
+        1. <ins class="block">If _nf_.[[RoundingType]] is ~morePrecision~, then</ins>
+          1. <ins class="block">Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"morePrecision"*).</ins>
+        1. <ins class="block">Else if _nf_.[[RoundingType]] is ~lessPrecision~, then</ins>
+          1. <ins class="block">Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"lessPrecision"*).</ins>
+        1. <ins class="block">Else,</ins>
+          1. <ins class="block">Perform ! CreateDataPropertyOrThrow(_options_, *"roundingPriority"*, *"auto"*).</ins>
```

Comment:
I don't think you need `class="block"` in this case, just `<ins></ins>` should suffice.

Reply:
I prefer `class="block"` for entire `<li>`s and that's what I use elsewhere in this doc.  I use `<ins>` for inline elements.